spec/spec_params.rb

```diff
@@ -1936,4 +1937,21 @@ module OptimizelySpec
   # SEND_FLAG_DECISIONS_DISABLED_CONFIG['sendFlagDecisions'] = false
 
   CONFIG_DICT_WITH_INTEGRATIONS_JSON = JSON.dump(CONFIG_DICT_WITH_INTEGRATIONS)
+
+  def self.deep_clone(obj)
```

Comment:
What is the purpose of cloning, is it an equivalent of python's deepcpy?

Reply:
That's exactly right. Allowing us to manually modify the datafiles in one test without effecting other tests.